lib/vmdb/deprecation.rb

```diff
@@ -30,7 +30,7 @@ def self.proc_for_default_log
       return unless default_log
       proc do |message, callstack|
         default_log.warn(message)
-        default_log.debug(callstack.join("\n  ")) if default_log.debug?
```

Comment:
WAT.  We were calling `debug` on the logger after first checking if it was debug?  We were trying to avoid the unnecessary string allocations in the case of info log level?  I vaguely recall you can pass a block to accomplish the same thing.

Reply:
in other words...

```
default_log.debug { callstack.join("\n  ") }
```

Or something like that... either way, I like the change... in dev and test mode, we don't care about the allocations.